__init__.py

```diff
@@ -89,32 +120,68 @@ def _format_email_body(self, diagnostics: dict) -> str:
                             json_str,
                             self.translate("email_signature")))
 
-    def _get_support_info(self, message: Message,
-                          profile: dict = None) -> dict:
+    def _check_service_status(self, message: Message = None) -> dict:
         """
-        Collect relevant information to include in a support ticket
-        :param message: Message associated with support request
+        Query services on the messagebus and report back their status
         """
-        user_profile = profile or get_user_prefs(message)
-        message_context = deepcopy(message.context)
+        message = message or Message("get_status")
 
         speech_module = self.bus.wait_for_response(
-            Message("mycroft.speech.is_ready"))
+            message.forward("mycroft.speech.is_ready"))
         speech_status = speech_module.data.get("status") if speech_module \
             else None
 
         audio_module = self.bus.wait_for_response(
-            Message("mycroft.audio.is_ready"))
+            message.forward("mycroft.audio.is_ready"))
         audio_status = audio_module.data.get("status") if audio_module \
             else None
 
         skills_module = self.bus.wait_for_response(
-            Message("mycroft.skills.is_ready"))
-        skills_module = skills_module.data.get("status") if skills_module \
+            message.forward("mycroft.skills.is_ready"))
+        skills_status = skills_module.data.get("status") if skills_module \
+            else None
+
+        gui_module = self.bus.wait_for_response(
+            message.forward("mycroft.gui_service.is_ready"))
+        gui_status = gui_module.data.get("status") if gui_module \
+            else None
+
+        enclosure_module = self.bus.wait_for_response(
+            message.forward("mycroft.PHAL.is_ready")
+        )
+        enclosure_status = enclosure_module.data.get("status") if enclosure_module \
             else None
 
+        admin_module = self.bus.wait_for_response(
+            message.forward("mycroft.PHAL.admin.is_ready")
+        )
+        admin_status = admin_module.data.get("status") if admin_module \
+            else None
+
+        return {"speech": speech_status,
+                "audio": audio_status,
+                "skills": skills_status,
+                "gui": gui_status,
+                "enclosure": enclosure_status,
+                "admin": admin_status}
+
+    def _get_log_files(self):
+        log_path = LOG.base_path
+        log_files = glob(join(log_path, "*.log"))
```

Comment:
Those should be limited in some form or another.
maybe to the last boot. That would require a signifier at bootup.

Reply:
Good point; I think I did parse files to a max number of lines in a previous version of this (using BASH scripts).

Reply:
Added truncation in #44